startup.sh

```diff
@@ -3,12 +3,12 @@
 
 export ACCESS_LOG_FORMAT='%(h)s %(l)s %({REMOTE_USER}i)s %(t)s "%(r)s" "%(q)s" %(s)s %(b)s %(D)s'
 
+export LCLS_NAMING_TOOL_TAXONS_DIR="$(pwd)/lcls_naming_tool/taxons"
+
 source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate
 
 cp ./lcls_naming_tool/lcls_naming_tool.py ./lcls_naming_tool/web
```

Comment:
I'm still remarkably confused about these cp steps

Reply:
The web server doesn't allow anything to be pip installed, so `pip install lcls_naming_tool` doesn't work and I couldn't figure out the right bash commands to use at the time, so I used `cp` to put a copy of the script in /web. I looked through some bash tutorials and created environmental variables based on Robert's `SCRIPT_DIR` example. This seems to be working now for the website.
```
export HOME_DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"

export PYTHONPATH="${HOME_DIR}/lcls_naming_tool"
```

Reply:
I like this! It's more portable and will continue to work if/as we expand the module, and won't result in extra files appearing at runtime.